managers/kafka/handler.py

```diff
@@ -0,0 +1,95 @@
+""" The implementation class for KafkaManager, the public interface used by main. """
+
+import asyncio
+
+from aiokafka.errors import KafkaError
+
+from kytos.core import log
+from napps.kytos.kafka_events.managers.kafka._producer import Producer
+from napps.kytos.kafka_events.managers.kafka._serializer import JSONSerializer
+from napps.kytos.kafka_events.managers.kafka.interface import KafkaManager
+from napps.kytos.kafka_events.settings import (
+    ACKS,
+    BATCH_SIZE,
+    BOOTSTRAP_SERVERS,
+    ENABLE_ITEMPOTENCE,
+    LINGER_MS,
+    MAX_REQUEST_SIZE,
+    TOPIC_NAME,
+)
+
+
+class KafkaDomainManager(KafkaManager):
```

Comment:
This can be a `KafkaManager` without the inherintance, see the comment below regarding indirections for now and the design decisions we agreed on.